plugins/bf4/bf4.cpp

```diff
@@ -0,0 +1,229 @@
+    }
+
+    if (squad == 0) {
+        oidentity << std::endl << "\"Squad\": \"No\""; // If squad value is 0, set squad state to "No" in identity
+    } else {
+        // If squad value is in a value between 1 and 26, set squad name in identity using NATO Phonetic alphabet
+        if (squad == 1)
+            oidentity << std::endl << "\"Squad\": \"Alpha\"";
+        else if (squad == 2)
+            oidentity << std::endl << "\"Squad\": \"Bravo\"";
+        else if (squad == 3)
+            oidentity << std::endl << "\"Squad\": \"Charlie\"";
+        else if (squad == 4)
+            oidentity << std::endl << "\"Squad\": \"Delta\"";
+        else if (squad == 5)
+            oidentity << std::endl << "\"Squad\": \"Echo\"";
+        else if (squad == 6)
+            oidentity << std::endl << "\"Squad\": \"Foxtrot\"";
+        else if (squad == 7)
+            oidentity << std::endl << "\"Squad\": \"Golf\"";
+        else if (squad == 8)
+            oidentity << std::endl << "\"Squad\": \"Hotel\"";
+        else if (squad == 9)
+            oidentity << std::endl << "\"Squad\": \"India\"";
+        else if (squad == 10)
+            oidentity << std::endl << "\"Squad\": \"Juliet\"";
+        else if (squad == 11)
+            oidentity << std::endl << "\"Squad\": \"Kilo\"";
+        else if (squad == 12)
+            oidentity << std::endl << "\"Squad\": \"Lima\"";
+        else if (squad == 13)
+            oidentity << std::endl << "\"Squad\": \"Mike\"";
+        else if (squad == 14)
+            oidentity << std::endl << "\"Squad\": \"November\"";
+        else if (squad == 15)
+            oidentity << std::endl << "\"Squad\": \"Oscar\"";
+        else if (squad == 16)
+            oidentity << std::endl << "\"Squad\": \"Papa\"";
+        else if (squad == 17)
+            oidentity << std::endl << "\"Squad\": \"Quebec\"";
+        else if (squad == 18)
+            oidentity << std::endl << "\"Squad\": \"Romeo\"";
+        else if (squad == 19)
+            oidentity << std::endl << "\"Squad\": \"Sierra\"";
+        else if (squad == 20)
+            oidentity << std::endl << "\"Squad\": \"Tango\"";
+        else if (squad == 21)
+            oidentity << std::endl << "\"Squad\": \"Uniform\"";
+        else if (squad == 22)
+            oidentity << std::endl << "\"Squad\": \"Victor\"";
+        else if (squad == 23)
+            oidentity << std::endl << "\"Squad\": \"Whiskey\"";
+        else if (squad == 24)
+            oidentity << std::endl << "\"Squad\": \"X-ray\"";
+        else if (squad == 25)
+            oidentity << std::endl << "\"Squad\": \"Yankee\"";
+        else if (squad == 26)
+            oidentity << std::endl << "\"Squad\": \"Zulu\"";
+
+        if (squad_leader)
+            oidentity << std::endl << "\"Squad leader\": \"Yes\""; // If squad leader value is 1, set squad leader state to "Yes" as squad leader in identity
```

Comment:
Should this be a boolean instead of a string?

So

`"Yes" -> true`

and

`"No" -> false`?